backend/app/modules/computation/Clusterizer.scala

```diff
@@ -0,0 +1,236 @@
+package modules.computation
+
+import java.awt.image.BufferedImage
+import java.io.File
+import javax.imageio.ImageIO
+
+import scala.annotation.tailrec
+import scala.collection.mutable.ListBuffer
+
+
+/**
+  * Utility class generating a cluster of points from an Image.
+  *
+  * This class read an RGB image, get channels to clusterize (e.g. Red only or
```

Comment:
reads
gets
average between multiple channels is not supported now
threshold
creates